src/EnergyPlus/General.cc

```diff
@@ -3219,7 +3224,6 @@ namespace General {
         static std::string VarDictOption2;
         //  LOGICAL,SAVE :: SchRpt = .FALSE.
         //  CHARACTER(len=MaxNameLength) :: SchRptOption
-        static bool GetReportInput(true);
```

Comment:
There are other statics in here, but they appear to be controlled fully by the value of this flag, so I pulled this one out.